### ISSUE TITLE
Script bug near Ice Caves *Probably going to be gone with next build I guess*

### ISSUE THREAD

**Issue by DaimeneX** [KitaenSilva/OSFOP-Project-Repo#26]
So, the comment about the cave being cold triggers ad infinitum from what I have seen.
I went for the pickaxe and it happened again.

**Comment by ShadowJonathan** [comment]
pics, pls

**Comment by DaimeneX** [comment]
Cant really picture that...
What I mean is, Niko will comment on it getting uncomfortably cold for them in the cave when we go towards it. *The one near squares*
Then we need to go back for the pickaxe (The one in the beginning of the mines) and once we are back in the area before ice cave Niko will AGAIN comment on it being cold.

Like the script repeats itself whenever entering the area from the door in the caves.

**Comment by ShadowJonathan** [comment]
ah

@KitaenSilva, your job

**Comment by DaimeneX** [comment]
Yup. It goes Ad Infinitum.
I just went for the bag to fix Amenity, went into area before ice cave and Niko AGAIN commented on it being cold.

**Comment by KitaenSilva** [comment]
Yup yup - that is because I used to have events being variable-based and inter-dependant. When I switched all the events before this one to be switch-based and independent, it borked all the ones that were still variable-based.

If you keep triggering the event, eventually it won't trigger anymore, and that's because at the end of each event it adds a +1 to the variable, which was what was supposed to prevent it from being triggered again.

It is a remnant of Kit learning how to do eventing!
